GVRf/Framework/framework/src/main/java/org/gearvrf/GVRTexture.java

```diff
@@ -215,6 +217,54 @@ void signal() {
             }
         }
     }
+    /**
+     *  Sets the name of the texture coordinate vertex
+     *  attribute to use for this texture and what shader
+     *  output is is assigned to.
+     *  This allows a texture to be used with any set
+     *  of texture coordinates from a mesh.
+     *  Initially this parameter is null. You may set it once
+     *  but then it cannot be changed afterwards.
+     *  This is because it is used to generate code for the
+     *  fragment shader and updating it would require the
+     *  shader to be recompiled.
+     * @param vertexTexCoord string with name of texture coordinate attribute in vertex.
+     * @param shaderVariable string with name of shader variable to get the texture coordinates.
+     * @throws UnsupportedOperationException if you change variables more than once.
+     */
+    public void setTexCoords(String vertexTexCoord, String shaderVariable) throws UnsupportedOperationException
```

Comment:
I think it would be better if these were ctor parameters. If you call this for the first time after the shader has been generated it is not going to have effect, right? Not something that should hold up the pull request.